modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -77,19 +74,28 @@
  * Partition command handler.
  */
 public class PartitionListener implements RaftGroupListener {
-    /** Partition storage. */
-    private final PartitionStorage storage;
+    /** Table ID. */
+    private final IgniteUuid tableId;
```

Comment:
Because tx manager requires a table id for acquring locks for tables. It seems ok for me to having tableId in PartitionListener, because each listener is bound to a table. Probably it can be refactored later. For now I would like to keep it as is.